web/src/filter.ts

```diff
@@ -442,7 +454,8 @@ export class Filter {
             if (term.operator === "") {
                 return term.operand;
             }
-            return sign + term.operator + ":" + Filter.encodeOperand(term.operand.toString());
+            const operator = util.canonicalize_stream_synonym(term.operator);
+            return sign + operator + ":" + Filter.encodeOperand(term.operand.toString());
```

Comment:
Would it be correct to always canonicalize operators here? I wonder if just calling `canonicalize_operator` here could be a good prep commit.

Reply:
The answer is no -- because we want to still generate `streams:` for the search box, OK.